zebra/label_manager.c

```diff
@@ -135,6 +151,11 @@ static int lm_zclient_read(struct thread *t)
 	/* read response and send it back */
 	ret = relay_response_back();
 
+	/* on error, schedule another read */
+	if ( ret == -1 )
```

Comment:
coding style: `if (ret == -1)` (2 spaces less)
(and maybe chain the 2 `if` with `&&`)

Reply:
thank you, the change was made by a colleague with that style, changing it, sorry :-)